src/components/common/delete-widget/delete-widget.directive.coffee

```diff
@@ -0,0 +1,20 @@
+module = angular.module('impac.components.common.delete-widget',[])
+
+module.directive('commonDeleteWidget', ($templateCache, ImpacWidgetsSvc, ImpacUtilities) ->
```

Comment:
For new components, we should be leveraging the angular `.component`. 

```
bindings: {
   # one-way binding
   parentWidget: '<' 
}
...

# lifecycle hooks
ctrl.$ngOnInit = ->
  ctrl.loading = false

```

`.directives` should really only be used now to attach functionality onto an element. For example the `impac-angular/src/components/widgets-common/autofocus/autofocus.directive.coffee` directive.